AmazonChimeSDK/AmazonChimeSDK/internal/video/DefaultVideoClient.swift

```diff
@@ -23,8 +23,12 @@ class DefaultVideoClient: VideoClient {
         switch logLevel.rawValue {
         case Constants.errorLevel, Constants.fatalLevel:
             logger.error(msg: msg)
+        case Constants.warningLevel, Constants.infoLevel:
```

Comment:
Should we print `Constants.debugLevel` here as well? Same for `DefaultAudioClient`.

Reply:
updated to simplify